onnxruntime/core/providers/webnn/builders/impl/squeeze_unsqueeze_op_builder.cc

```diff
@@ -78,15 +78,25 @@ Status SqueezeUnsqueezeOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_buil
     }
   }
 
-  if (axes_data.size() > 0) {
-    options.set("axes", emscripten::val::array(axes_data));
-  }
-
   emscripten::val output = emscripten::val::undefined();
   if (op_type == "Squeeze") {
+    if (axes_data.size() > 0) {
+      options.set("axes", emscripten::val::array(axes_data));
+    }
     output = model_builder.GetBuilder().call<emscripten::val>("squeeze", input, options);
```

Comment:
I presume `squeeze` -> `reshape` will be handled in a later CR.

Reply:
Yep, and also `flatten2d`. I should merge them, but another CR is fine.